src/applications/disability-benefits/2346/containers/App.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
```

Comment:
Can we have rename this to something about more semantic? I think the remainder of the vets-website repo only has one App.js.

Reply:
> Can we rename this to something more semantic? I think the remainder of the vets-website repo only has one App.js.

This was created by the form generator as well.  It's only being imported into app-entry.jsx. Do you think we should delete this file and move the functionality to app-entry.jsx ?